repairnator/repairnator-checkbranches/src/main/java/fr/inria/spirals/repairnator/checkbranches/Launcher.java

```diff
@@ -45,128 +45,32 @@ private void defineArgs() throws JSAPException {
         // Verbose output
         this.jsap = new JSAP();
 
-        // help
-        Switch sw1 = new Switch("help");
-        sw1.setShortFlag('h');
-        sw1.setLongFlag("help");
-        sw1.setDefault("false");
-        this.jsap.registerParameter(sw1);
-
-        // verbosity
-        sw1 = new Switch("debug");
-        sw1.setShortFlag('d');
-        sw1.setLongFlag("debug");
-        sw1.setDefault("false");
-        this.jsap.registerParameter(sw1);
-
-        sw1 = new Switch("skipDelete");
-        sw1.setLongFlag("skipDelete");
-        sw1.setDefault("false");
-        sw1.setHelp("Skip the deletion of docker container.");
-        this.jsap.registerParameter(sw1);
-
-        sw1 = new Switch("notifyEndProcess");
-        sw1.setLongFlag("notifyEndProcess");
-        sw1.setDefault("false");
-        sw1.setHelp("Activate the notification when the process ends.");
-        this.jsap.registerParameter(sw1);
-
-        sw1 = new Switch("humanPatch");
+        this.jsap.registerParameter(LauncherUtils.defineArgHelp());
+        this.jsap.registerParameter(LauncherUtils.defineArgDebug());
+        this.jsap.registerParameter(LauncherUtils.defineArgRunId());
+        this.jsap.registerParameter(LauncherUtils.defineArgInput("Specify the input file containing the list of branches to reproduce"));
+        this.jsap.registerParameter(LauncherUtils.defineArgOutput(true, false, true, false, "Specify where to put output data"));
```

Comment:
This line is really not understandable when reading it...

Reply:
I did some changes in the method `defineArgOutput`. Please, tell me if it is still bad.